libnavui-maneuver/src/main/res/layout-land/mapbox_maneuver_layout.xml

```diff
@@ -57,7 +57,7 @@
             <com.mapbox.navigation.ui.maneuver.view.MapboxManeuversList
                 android:id="@+id/upcomingManeuverRecycler"
                 android:layout_width="0dp"
-                android:layout_height="200dp"
+                android:layout_height="150dp"
```

Comment:
How's this solving the issue? I was actually referring to the `width` not the `height` 😅 

![test-app_hardcoded_list_size](https://user-images.githubusercontent.com/1668582/110710669-4a03b300-81cc-11eb-9f44-f65282b4bcd3.png)

In any case, hardcoding the value here won't probably work for other devices with different screen sizes, etc.